src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -133,37 +90,43 @@ export const TranscriptsListItemInfo = (
       <div className={midStyles}>
         <div className={styles.topLeft}>
           <div>
-            <strong>{transcript.biotype}</strong>
+            <strong>{transcript.so_term}</strong>
           </div>
           <div>{getTranscriptLocation()}</div>
         </div>
         <div className={styles.topMiddle}>
-          {transcript.cds && (
+          {isProteinCodingTranscript(transcript) && (
             <>
               <div>
-                <strong>{getAminoAcidLength()} aa</strong>
+                <strong>{getProductAminoAcidLength(transcript)} aa</strong>
               </div>
               <div>ENSP1000000000</div>
```

Comment:
Is this something we still don't have the data available yet?

Reply:
✔️